### ISSUE TITLE
Unity crashes on MacOS

### ISSUE THREAD

**Issue by kotrs** [lagadic/visp_unity#18]
I generated ViSPUnity.dylib in Xcode. I added it to Assets folder. I renamed it to ViSPUnity.bundle.
Unity crashes when I start mbt or ar scene. I have Unity 2018.3.14

I tested to compile an example blob and test-blob-opencv runs ok.

**Comment by fspindle** [comment]
Same for me

I tried to modify the `CMakeLists.txt` file to create a bundle but without more success:
```
project(ViSPUnity)

cmake_minimum_required(VERSION 2.6)

if(APPLE)
  set(CMAKE_MACOSX_RPATH ON)
endif()

find_package(VISP REQUIRED visp_core visp_io visp_detection visp_mbt)

include_directories(${VISP_INCLUDE_DIRS})

add_library(ViSPUnity SHARED ViSPUnity.cpp ViSPUnity.h)
target_link_libraries(ViSPUnity ${VISP_LIBRARIES})

if(APPLE)
  set_target_properties(ViSPUnity PROPERTIES BUNDLE TRUE)
  set_target_properties(ViSPUnity PROPERTIES SUFFIX ".bundle")
endif()
```

From what I have seen on the web, the solution could be to use Xcode instead of CMake

**Comment by fspindle** [comment]
@kotrs I was able to reproduce this issue. From what I found on the web it comes from MacOS camera authorization requested before usage in Unity.

The work around that I found is to quit Unity Hub and launch it from a terminal
```
$ /Applications/Unity\ Hub.app/Contents/MacOS/Unity\ Hub
```
If this is not enough, try to quit properly your opened applications and restart your computer.

Note also that there is also an other issue with the `webCamTexture` that doesn't update width and height according to the image size. This issue is fixed in PR #21 with a lot of other improvements.

I would appreciate if you can test this PR and let me know.
